core/domain/role_services.py

```diff
@@ -96,6 +100,9 @@
 ]
 
 # Users can be viewed by following list of role IDs via admin interface.
+#
+# NOTE: Do not include LEARNER role in this list as it does not represent
+#   role for a separate user account, rather a profile within the account.
```

Comment:
Nit: ``role for a separate user account, but rather a profile within the account.``

Reply:
Done.